modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|NodeSelector is the simplest recommended form of node selection constraint.
+|type: object, additionalProperties: type: string
 |
+
+|`replicas`
+| Replicas represents the number of replicas to create for this component.
+| type: integer, format: int32
+|
+
+|`tolerations`
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items)
+|The pod this Toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
```

Comment:
```suggestion
|The pod this toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
```

Reply:
```suggestion
|The pod to which the tolerations are attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
```

Switched from singular to plural form, as this property defines a list of tolerations.